connectivity/drivers/lora/COMPONENT_SX126X/mbed_lib.json

```diff
@@ -1,6 +1,10 @@
 {
     "name": "SX126X-lora-driver",
     "config": {
+        "radio": {
```

Comment:
If I understand this correctly this provides the override of "lora.radio"

Reply:
yes, I don't find any clean way to override some config value defined in another lib....

Reply:
That's a longstanding issue 😞 .